src/js/nav.js

```diff
@@ -1,107 +1,118 @@
-/*global require*/
 /**
  * Add a second navigation menu to quickly navigate to
  * anchors in the page.
+ *
  */
+
 'use strict';
 
-var $ = require('jquery');
-var getCurrentLayout = require('o-grid').getCurrentLayout;
+var oViewport = require('o-viewport');
+oViewport.listenTo('scroll');
+oViewport.listenTo('resize');
 
-$(function() {
-	var list = '', lis = [], scrollmargin, scrolltimer, resizetimer, dockpoint, dockmargin, headings = [], currentheading;
+document.addEventListener('o.DOMContentLoaded', function() {
```

Comment:
why not listen just once in the main file, and then execute all the other modules from there? In the current state, how do you run the code without dispatching the `o.DOMContentLoaded` event?